frontend/src/properties/propertysections/LinkPropertySection.tsx

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import Link from '@material-ui/core/Link';
+import { LinkPropertySectionProps } from 'properties/propertysections/LinkPropertySection.types';
+import React from 'react';
+
+/**
+ * Defines the content of a Link property section.
+ */
+export const LinkPropertySection = ({   widget }: LinkPropertySectionProps) => {
```

Comment:
Are you using the Prettier formatter in VSCode? the formatting of the déstructuration of the props into `{   widget }` feels odd